scripts/processMarkdownTable.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+function processMarkdownTable(dataJson) {
+	let table = `
+|Package|Size|
+|----|----|`;
+
+	Object.keys(dataJson).forEach(package => {
```

Comment:
As you're using Node 10, I think you can safely use `Object.entries` here.

I'd avoid using `package` as an identifier though because it is [a reserved word](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Keywords), at least in strict mode.

ie. `Object.entries(dataJson).forEach(([packageName, size]) => { ... }`